gonzalo-salazar/tarea-1/index.css

```diff
@@ -0,0 +1,111 @@
+body {
+    background-image: url("img/fondo.jpg");
+    background-size: cover;
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    background-image: url("img/nba.webp");
+    background-size: cover;
+    background-position: center;
+    height: 130vh;
+    margin: 0px auto;
+    width: 55%;
+    margin-top: 50px;
+    box-shadow: 0px 0px 9px 0px #000000;
+    margin-bottom: 50px;
+    border-radius: 15px;
+}
+
+.titulo_1 {
```

Comment:
@Gonza0624 , make sure you use english as main language for your code

Reply:
I already made the corrections